src/main/java/cz/fi/muni/pa165/esports/entity/Player.java

```diff
@@ -1,4 +1,71 @@
 package cz.fi.muni.pa165.esports.entity;
 
+import cz.fi.muni.pa165.esports.enums.Gender;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+/**
+ * Player entity
+ * @author Radovan Tomasik
+ */
+@Entity
 public class Player {
+    @Column(nullable = false)
+    private String name;
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long player_id;
+    private Integer age;
+
+    @Enumerated(EnumType.STRING)
+    private Gender gender;
+
```

Comment:
I think you're missing a constructor with `id` as parameter (to make ID autogeneration work properly?).

```suggestion

    public Player(Long id) {
        this.player_id = id
    }

```

Reply:
done